src/Contexts/Adaptors/SubstrateDestinationAdaptor.tsx

```diff
@@ -121,19 +118,15 @@ export const SubstrateDestinationAdaptorProvider = ({
           }
         });
       });
-      setListenerActive(unsubscribe);
+      setListenerActive(true);
     } else if (listenerActive && !depositNonce) {
-      const unsubscribeCall = async () => {
-        setListenerActive(undefined);
-      };
-      unsubscribeCall();
```

Comment:
Why we don't need it more?

Reply:
As discussed `listenerActive` is not called in the code.